resources/views/components/landing/cta-invite.blade.php

```diff
@@ -0,0 +1,14 @@
+<div class="h-[600px] w-full flex-col bg-[url('https://s3-alpha-sig.figma.com/img/2f1a/60a1/6f1444bf3b2a06961f5ff319c0a9aaa0?Expires=1737936000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=oHpz7iAovz9CW8FZFFge2sJ2kclj7lZMAPqi-Y0MCNNeU6KXTthimd8Ss3dcBfoWbjsXtFW1UO9~xJ7gtgbWkttQG2V0ElJme-ss7rEtFUXpDSH3gxI~54XFpRlBzWub3L3r0NH1A7EfxRoIcqwO6herqxnRbVRAbpCDppS-X8bycYA8~XGcT5kOeco67JCIR0h5DmrMnejv~jHByaVDj5DHb5G2XII3cudXe9AvGRdZX4XKhtjpQS7KaHN0GCpuxPk~3CsM1c-i9ys3H9pfQLdMh4plqr7C2GxdEUeWMAFFGFdso7GNseZYk5m1ZYoZ4Rs2VgJcYzW5QplVJFtb7A__')] bg-cover bg-center relative">
```

Comment:
Relevant to storage folder. Please, download the image and "minify" it before uploading to the project.

To sync the `./storage/app/public` with `./public/storage/` please run locally:

```
php artisan storage:link
```